src/Control/Monad/Linear.hs

```diff
@@ -35,9 +58,46 @@ import Data.Unrestricted.Linear
 import Prelude.Linear.Internal.Simple ((.), ($))
 import qualified Data.Functor.Linear.Internal as Data
 
+
+-- $stateT
```

Comment:
I'd put this paragraph close to `StateT` in the file. But, otherwise: ok.

Reply:
StateT is defined in another file that's internal so I can't migrate it.